api/src/main/java/au/org/aodn/nrmn/restapi/repository/projections/SpeciesWithAttributesCsvRow.java

```diff
@@ -0,0 +1,23 @@
+package au.org.aodn.nrmn.restapi.repository.projections;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class SpeciesWithAttributesCsvRow {
+
+    private String letterCode;
+
```

Comment:
i think you can implement  UiSpeciesAttributes and remove all the duplicate fields

Reply:
Looked into this and it could be done, but not without substantial changes to one or the other usage. Also the CsvRow has the lettercode which I think makes it clearer to keep seperate.